sdk/scheduler/src/main/java/com/mesosphere/sdk/offer/evaluate/ResourceEvaluationStage.java

```diff
@@ -150,9 +151,10 @@ public EvaluationOutcome evaluate(MesosResourcePool mesosResourcePool, PodInfoBu
 
         return pass(
                 this,
-                offerRecommendation == null ? null : Arrays.asList(offerRecommendation),
-                "Offer contains sufficient '%s'",
-                resourceRequirement.getName());
+                offerRecommendation == null ? Collections.emptyList() : Arrays.asList(offerRecommendation),
+                "Offer contains sufficient '%s': requirement=%s",
+                resourceRequirement.getName(),
+                TextFormat.shortDebugString(resourceRequirement.getValue()));
```

Comment:
Alternately the evaluation could store the logged info to a buffer instead of logging directly, then logged as a block at the end of the evaluation cycle. Somewhat exceeding the scope of this PR though.